library/src/com/google/maps/android/geojson/GeoJsonLineString.java

```diff
@@ -0,0 +1,54 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.ArrayList;
+
+/**
+ * A GeoJsonLineString geometry contains a number of {@link com.google.android.gms.maps.model.LatLng}s.
+ */
+public class GeoJsonLineString implements GeoJsonGeometry {
+
+    private final static String GEOMETRY_TYPE = "LineString";
+
+    private final ArrayList<LatLng> mCoordinates;
+
+    /**
+     * Creates a new GeoJsonLineString object
+     *
+     * @param coordinates array of coordinates of GeoJsonLineString to store
+     */
+    public GeoJsonLineString(ArrayList<LatLng> coordinates) {
```

Comment:
As above, does this need to be an ArrayList? If List or Collection would do, use them instead.